thavalon-server/src/game/snapshot.rs

```diff
@@ -0,0 +1,308 @@
+//! Snapshots of the current state of a game. These can be persisted or sent to clients.
```

Comment:
I think this got pulled in from how I set up branches. If it's alright with you, I'll leave it here and delete the other branch.

Reply:
Fine with me, looks like this file is slightly different than other PR is all